src/Models/ShallowWaterModels/shallow_water_advection_operators.jl

```diff
@@ -52,11 +67,14 @@ Calculates the divergence of the mass flux into a cell,
 
 which will end up at the location `ccc`.
 """
-@inline function div_Uh(i, j, k, grid, solution)
-    1/Vᶜᶜᶜ(i, j, k, grid) * (δxᶜᵃᵃ(i, j, k, grid, Ax_qᶠᶜᶜ, solution.uh) + 
-                             δyᵃᶜᵃ(i, j, k, grid, Ay_qᶜᶠᶜ, solution.vh))
+@inline function div_Uh(i, j, k, grid, advection, solution, formulation)
+    1/Azᶜᶜᶜ(i, j, k, grid) * (δxᶜᵃᵃ(i, j, k, grid, Ax_qᶠᶜᶜ, solution[1]) + 
+                              δyᵃᶜᵃ(i, j, k, grid, Ay_qᶜᶠᶜ, solution[2]))
```

Comment:
```suggestion
    1/Azᶜᶜᶜ(i, j, k, grid) * (δxᶜᵃᵃ(i, j, k, grid, Δy_qᶠᶜᶜ, solution[1]) + 
                              δyᵃᶜᵃ(i, j, k, grid, Δx_qᶜᶠᶜ, solution[2]))
```

more robust in case Δz is not 1?